app/panel/components/Panel.jsx

```diff
@@ -211,24 +217,36 @@ class Panel extends React.Component {
 		});
 	};
 
-	_handlePromoSelectBasicClick = () => {
-		// TODO send metrics ping
+	_handlePromoSelectBasicClick = (location) => {
 		this.props.actions.togglePromoModal();
+
+		sendMessage('ping', 'promo_modals_select_basic_panel');
 	};
 
-	_handlePromoSelectPlusClick = () => {
-		// TODO send metrics ping
+	_handlePromoSelectPlusClick = (location) => {
```

Comment:
Does ESLint throw an unused arg error here?

Reply:
Thought I needed this, realized I didn't, but forgot to remove. Removing. Also removed unused modal arg from _handlePromoSignInClick